frontend/src/console/components/NoticeBoard/styles.scss

```diff
@@ -0,0 +1,110 @@
+@import '~styles/theme';
+
+.deployment-alert {
+  .healthy,
+  .changes_pending {
+    background-color: $light-blue;
```

Comment:
Instead of overriding colours like this, use the classes in jsx. i.e.
bg-primary, text-primary etc.

Reply:
@xitij2000 
I can't find class alternatives for some colors, like this one (`$light-blue`), because they defined in our code (`frontend/src/styles/_theme.scss`).

So I'm not sure how to fix this.

Reply:
@xitij2000 Yay! I managed to remove almost all color overriding. Thanks for that suggestion.